TodoWithGenericControllers/TodoWithGenericControllers.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+	</PropertyGroup>
+	<ItemGroup Label="Project Packages">
```

Comment:
Why is there an item group with a label? Why are the package references not aligned? Is this using tabs?

Reply:
> Why is there an item group with a label? 
I should remove it. It is just habit to group package or do something on msbuild process.

> Why are the package references not aligned? Is this using tabs?
Yup, it using the Tabs. Fix now.